go/protocols/materialize/materialize.proto

```diff
@@ -84,6 +87,36 @@ service Driver {
       returns (stream TransactionResponse);
 }
 
+// SpecRequest is the request type of the Spec RPC.
+message SpecRequest {
+  // Endpoint type addressed by this request.
+  flow.EndpointType endpoint_type = 1;
+  // Driver specification, as an encoded JSON object.
+  // This may be a partial specification (for example, a Docker image),
+  // providing only enough information to fetch the remainder of the
+  // specification schema.
+  string endpoint_spec_json = 2 [
+    (gogoproto.casttype) = "encoding/json.RawMessage",
+    json_name = "endpointSpec"
+  ];
+}
+
+// SpecResponse is the response type of the Spec RPC.
+message SpecResponse {
+  // JSON schema of a complete endpoint specification.
+  string endpoint_spec_schema_json = 1 [
+    (gogoproto.casttype) = "encoding/json.RawMessage",
+    json_name = "endpointchema"
+  ];
+  // JSON schema of a complete endpoint specification.
+  string resource_spec_schema_json = 2 [
+    (gogoproto.casttype) = "encoding/json.RawMessage",
+    json_name = "endpointchema"
```

Comment:
Similar typos here as with `capture.proto`.

Reply:
Thanks. Same story -- I'll address in `protocols` repo